README.md

```diff
@@ -39,20 +39,20 @@ _Note: This module is for Nuxt 3. [Check out `@storyblok/nuxt-2` for Nuxt 2](htt
 
 ### Installation
 
-Install `@storyblok/nuxt`:
+Install `@storyblok/nuxt` and `axios`:
```

Comment:
We're about to remove axios, and Node 16+ will install peer deps automatically so we made the decision to keep axios out of docs for now. Could you remove it?

Reply:
Sure, will do